src/jit/codegencommon.cpp

```diff
@@ -10461,38 +10461,24 @@ unsigned CodeGen::getFirstArgWithStackSlot()
 {
 #if defined(UNIX_AMD64_ABI) || defined(_TARGET_ARMARCH_)
     unsigned baseVarNum = 0;
-#if defined(FEATURE_UNIX_AMR64_STRUCT_PASSING)
-    baseVarNum = compiler->lvaFirstStackIncomingArgNum;
-
-    if (compiler->lvaFirstStackIncomingArgNum != BAD_VAR_NUM)
+    // Iterate over all the local variables in the Lcl var table.
```

Comment:
Is it me or this comment is wrong? The for loop below seems to iterate only over the args.

Reply:
You're correct - the comment is wrong. In my defense, it isn't new (though the diffs seem to think so), it was just moved from below.
I'll correct it.